app/portainer/views/account/accountController.js

```diff
@@ -100,7 +111,9 @@ angular.module('portainer.app').controller('AccountController', [
       const state = StateManager.getState();
       const userDetails = Authentication.getUserDetails();
       $scope.userID = userDetails.ID;
+      $scope.userRole = Authentication.getUserDetails().role;
       $scope.forceChangePassword = userDetails.forceChangePassword;
+      $scope.timesPasswordChangeSkipped = StateManager.getState().UI.timesPasswordChangeSkipped || 0;
```

Comment:
state is already initialized by `StateManager.getState()`

```suggestion
      $scope.timesPasswordChangeSkipped = state.UI.timesPasswordChangeSkipped || 0;
```

Reply:
ah missed one, good catch